Makefile

```diff
@@ -86,137 +127,179 @@ clean_templates:
 	@rm -rf templates/springboot/http/target
 	@rm -f templates/**/.DS_Store
 
-.PHONY: generate/zz_filesystem_generated.go
 
-generate/zz_filesystem_generated.go: clean_templates
+# the filesysetm file itself is defined as being fresh if all the files in
+# templates are fresh as well as the certificates file.
+generate/zz_filesystem_generated.go: $(TEMPLATES) templates/certs/ca-certificates.crt
```

Comment:
I very much on purpose remove similar condition in the past. This can detect that file was modified but it won't for instance detect file deletion.

Reply:
This should be phony.

Reply:
The `generate/zz_filesystem_generated.go` recipe takes 200ms. Most of which is actually spend on `go run`'s on the fly compilation. The actual template generation takes 40ms. I don't see why we cannot run it all the time.

Reply:
I have removed all but the minimal makefile updates and will submit them separately for discussion.